.circleci/config.yml

```diff
@@ -0,0 +1,103 @@
+# CircleCI jobs are only enabled to on Pull Requests and commits to master branch.
+# "Only build pull requests" enabled in Project's Advanced Settings.
+version: 2.1
+jobs:
+  build_test:
+    docker:
+      - image: cimg/python:3.6
```

Comment:
python 3.6 is getting close to EOL, do we want to standardize on python 3.8 or something?

Reply:
We updated our SDK's min requirement from [py2.7 to py3.6](https://github.com/scaleapi/scaleapi-python-client/blob/master/setup.py#L45) and wanted to be still compatible while py3.6 is still maintained. CircleCI is matching that min requirement to be consistent.